Makefile

```diff
@@ -3,55 +3,63 @@ DEFAULT: build
+	$(GO) build -o $(APP) -ldflags "-X 'main.BuildTime=$(BUILDTIME)' -X main.GitCommit=$(GITCOMMIT) -X main.Version=$(PROGVER)"
 
 rpm:
 	mkdir -p ./.ignore/SOURCES
 
 	# CPE service 
-	tar -czf ./.ignore/SOURCES/cpe_themis-$(PROGVER).tar.gz --transform 's/^\./cpe_themis-$(PROGVER)/' --exclude ./.git --exclude ./.ignore --exclude ./conf --exclude ./deploy --exclude ./vendor --exclude ./vendor .
+	tar -czf ./.ignore/SOURCES/cpe_themis-$(RPM_VERSION)-$(RPM_RELEASE).tar.gz --transform 's/^\./cpe_themis-$(RPM_VERSION)-$(RPM_RELEASE)/' --exclude ./.git --exclude ./.ignore --exclude ./conf --exclude ./deploy --exclude ./vendor --exclude ./vendor .
 	cp conf/cpe_themis.service ./.ignore/SOURCES
 	cp themis.yaml  ./.ignore/SOURCES/cpe_themis.yaml
 
 	# RBL service
-	tar -czf ./.ignore/SOURCES/rbl_themis-$(PROGVER).tar.gz --transform 's/^\./rbl_themis-$(PROGVER)/' --exclude ./.git --exclude ./.ignore --exclude ./conf --exclude ./deploy --exclude ./vendor --exclude ./vendor .
+	tar -czf ./.ignore/SOURCES/rbl_themis-$(RPM_VERSION)-$(RPM_RELEASE).tar.gz --transform 's/^\./rbl_themis-$(RPM_VERSION)-$(RPM_RELEASE)/' --exclude ./.git --exclude ./.ignore --exclude ./conf --exclude ./deploy --exclude ./vendor --exclude ./vendor .
 	cp conf/rbl_themis.service ./.ignore/SOURCES
 	cp themis.yaml  ./.ignore/SOURCES/rbl_themis.yaml
 
 	# Standalone-mode service - All other XMiDT services are setup this way
-	tar -czf ./.ignore/SOURCES/$(APP)-$(PROGVER).tar.gz --transform 's/^\./$(APP)-$(PROGVER)/' --exclude ./.git --exclude ./.ignore --exclude ./conf --exclude ./deploy --exclude ./vendor --exclude ./vendor .
+	tar -czf ./.ignore/SOURCES/$(APP)-$(RPM_VERSION)-$(RPM_RELEASE).tar.gz --transform 's/^\./$(APP)-$(RPM_VERSION)-$(RPM_RELEASE)/' --exclude ./.git --exclude ./.ignore --exclude ./conf --exclude ./deploy --exclude ./vendor --exclude ./vendor .
 	cp conf/themis.service ./.ignore/SOURCES
 	cp themis.yaml  ./.ignore/SOURCES
 
 	cp LICENSE ./.ignore/SOURCES
 	cp NOTICE ./.ignore/SOURCES
 	cp CHANGELOG.md ./.ignore/SOURCES
 
+	# CPE service
 	rpmbuild --define "_topdir $(CURDIR)/.ignore" \
-    		--define "_version $(PROGVER)" \
-    		--define "_release 1" \
+    		--define "_version $(RPM_VERSION)" \
+    		--define "_release $(RPM_RELEASE)" \
     		-ba deploy/packaging/cpe_themis.spec
 
+	# RBL service
 	rpmbuild --define "_topdir $(CURDIR)/.ignore" \
-     		--define "_version $(PROGVER)" \
-     		--define "_release 1" \
-     		-ba deploy/packaging/rbl_themis.spec
+    		--define "_version $(RPM_VERSION)" \
+    		--define "_release $(RPM_RELEASE)" \
+    		-ba deploy/packaging/rbl_themis.spec
 
+	# Standalone-mode service - All other XMiDT services are setup this way
 	rpmbuild --define "_topdir $(CURDIR)/.ignore" \
-     		--define "_version $(PROGVER)" \
-     		--define "_release 1" \
-     		-ba deploy/packaging/$(APP).spec
+			--define "_version $(RPM_VERSION)" \
+			--define "_release $(RPM_RELEASE)" \
+			-ba deploy/packaging/$(APP).spec
```

Comment:
Double check that the spacing doesn't crash stuff (I'm not familiar enough with makefiles wrt this)